network-api/networkapi/wagtailpages/factory/blog.py

```diff
@@ -49,15 +49,17 @@ def generate(seed):
         print('blog namespace exists')
     except WagtailPage.DoesNotExist:
         print('Generating a blog namespace')
-        blog_namespace = MiniSiteNamespaceFactory.create(
+        blog_namespace = IndexPageFactory.create(
             parent=home_page,
             title='blog',
-            live=False
+            header='blog',
+            live=True
         )
 
-    try:
-        BlogPage.objects.get(title='post')
-        print('a post page (BlogPage) exists')
-    except BlogPage.DoesNotExist:
-        print('Generating a post page (BlogPage) under namespace')
-        BlogPageFactory.create(parent=blog_namespace, title='post')
+    print('Generating blog posts under namespace')
+    for i in range(7):
+        title = f'post {i}'
```

Comment:
Let's add a few random words to this so it's more representative of title length on live blog posts.